lorien/UI/Dialogs/AddKeyDialog.gd

```diff
@@ -0,0 +1,64 @@
+extends WindowDialog
+
+# -------------------------------------------------------------------------------------------------
+export var action_name: String
+export var readable_action_name: String
+
+# -------------------------------------------------------------------------------------------------
+var _pending_bind_event = null
+
+# -------------------------------------------------------------------------------------------------
+func _process(delta) -> void:
+	$VBoxContainer/EventText.text = tr(
+		"KEYBINDING_DIALOG_BIND_ACTION"
+	).format({"action": readable_action_name})
```

Comment:
Does this need to happen every frame? I don't mind that much, but getting the node should be cached at least (in an onready variable instead of getting it every frame - which has a cost)

Reply:
It's fine, you don't have to overengineer this. Just cache the label in in an onready variable.

Reply:
Meh... that would be buggy because then the "live language switching" feature would break yet again. The KEYBINDING_DIALOG_BIND_ACTION needs to be reloaded every time the language changes. I will just go the extra mile and implement it properly :-)